ldap-ui/src/main/resources/LDAP/Code/Translations.da.xml

```diff
@@ -39,8 +39,7 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content>## XWiki Translation for language da generated from http://www.l10n.xwiki.org
```

Comment:
I don't think weblate should strip comments... Looks like a bug.

Reply:
Note that in this case we could remove this comment since we're supposed to be bi-directional now but still it means there's a problem and it shouldn't be weblate's job to remove them!

Reply:
Well translations are not really supposed to contains comments (other than what what generated by the translation tool itself) so I don't really think this is a big issue.

Reply:
It's not an issue is non bi-directional setup but our is bi-directional now

Reply:
and even if we were to decide to go back to mono directional again it's even more important to add back this comment to show that committers should not change those translations.

Reply:
@vmassol I think we can merge that as we took the decision to keep the same structure as the base translation file (english).